src/media/js/main.js

```diff
@@ -160,8 +162,10 @@ function(_) {
 
 
             var manifest_url = settings.manifest_url;
-            // Note: only the iframed app defines a manifestURL for now.
-            if (manifest_url) {
+            // We want to check for an update, so we need the manifest url. In
+            // addition, we only want to show the update notification banner if
+            // we are inside an app (not just browsing the website).
+            if (manifest_url && (is_packaged_app || is_iframed_app)) {
```

Comment:
I'm confused by https://github.com/mozilla/fireplace/blob/master/src/media/js/settings_local_hosted.js#L6 and https://github.com/mozilla/fireplace/blob/master/package/templates/settings_local_package.js#L11-L13